HHS/OPRE-OPS
# Ops 441 initial page and combobox

## What changed

- Added `User Admin` page (only accessible by users with the `USER_ADMIN` role)
- Added `Budget Team` user with roles `BUDGET_TEAM` and `USER_ADMIN`
- Updated `UserInfo` component to make editable by `USER_ADMIN` (and display human readable values)
- Added new RTKQ API `opsAuthAPI`
- Improvements/fixes to `msw` JS stub (`mocks.js`)

## Issue

https://github.com/HHS/OPRE-OPS/issues/441

## How to test

1. Testing that only `USER_ADMIN` role can access the `User Admin` menu option

- Login as `Admin` and the menu option should not be visible.
- Login as `Budget Team` and the option should be visible (since BT is a `USER_ADMIN`)

2. Testing that only `USER_ADMIN` has an editable view of `UserInfo`

- Login as `Admin` (or any user other than `Budget Team`)
- Click on the user's email address in the upper right of the page to go to the `User Details` page.
- Verify that the `UserInfo` is read-only.

- Login as `Budget Team`
- Click on the user's email address in the upper right of the page to go to the `User Details` page.
- Verify that the `UserInfo` is editable, i.e. has combo boxes for status, role, and division

N.B. The actual persistence of the `UserInfo` changes will be covered in a future PR.   (There is already too much going on in this PR)

## Screenshots

_If relevant, e.g. for a front-end feature_

## Definition of Done Checklist
